esm/shared/parse-from-string.js

```diff
@@ -57,7 +57,12 @@ export const parseFromString = (document, isHTML, markupLanguage) => {
     onopentag(name, attributes) {
       let create = true;
       if (isHTML) {
-        if (ownerSVGElement) {
+        if (name === 'body') {
+          // Accessing the document.body property will create the <body>
+          node = document.body;
+          create = false;
+        }
```

Comment:
The HTML spec says "The body element of a document is the first of [the `html` element](https://html.spec.whatwg.org/multipage/dom.html#the-html-element-2)'s children that is either a [`body`](https://html.spec.whatwg.org/multipage/sections.html#the-body-element) element or a [`frameset`](https://html.spec.whatwg.org/multipage/obsolete.html#frameset) element, or null if there is no such element." -- https://html.spec.whatwg.org/multipage/dom.html#dom-document-body-dev

But in terms of multiple `<body>` and how it behaves, the browser seems to resolve to the same `Node`.

```js
new DOMParser().parseFromString('<html><body>foo</body><body>bar</body></html>', "text/html").documentElement.outerHTML
// '<html><head></head><body>foobar</body></html>'
```

For reference, `linkedom` and the browser both behave the same here ✅

Reply:
still it's no goal of this project to care about these shenanigans ... but good to know, thanks

Reply:
In this case, it's the simplest way to support and turns out to behave identically 👌